FLEDGE.md

```diff
@@ -1184,3 +1187,16 @@ All `Ad-Auction-Additional-Bid` response headers are intercepted by the browser
 #### 6.4 Reporting Additional Bid Wins
 
 For additional bids that win the auction, event-level win reporting is supported, just as it is for bids generated from stored interest groups. However, additional bids have distinct security concerns. The browser can guarantee that bids generated from stored interest groups were, in fact, generated by the buyer's origin, but it cannot provide this same guarantee for additional bids. An additional bid can only win if none of the negative interest groups specified by the additional bid are present on the user's device, and so the signature validation described above in section [6.2.3 Additional Bid Keys](#623-additional-bid-keys) also can't help with this. To ensure that buyers understand that they're being asked to report on an additional bid, which cannot be verified having been generated by the buyer's origin, the reporting for additional bids is done by calling reporting functions with a slightly different name. Instead of `reportWin()`, the browser instead calls a function named `reportAdditionalBidWin()`to report a winning additional bid.
+
+### 7. Debugging Extensions
+
+#### 7.1.1 Downsampling
+
+This design has two main goals:
+*   Prevent sending fDO reports very often to protect user privacy:
+Only sending reports 1/1000 times the forDebuggingOnly API is called, and
+If a report is sent, “lock-out” all adtechs out of sending a report for 3 years, and
+If an adtech calls the API, put them in a  “cool-down” period where calls to the API by that given adtech are not able to send reports.
+*  Prevent adtechs who accidentally (e.g. due to a bug in their code) call the API repeatedly for all users, from locking themselves out of sending any more reports for years. This is accomplished by 90% of the time putting that adtech in a 2 week “cooldown” period, and only 10% of the time putting that adtech in a 1 year cooldown period.
```

Comment:
maybe useful to add a link to the github issue for how we calculated the numbers?

Reply:
Sure, perhaps you could add a sentence at the end like "See [issue 632](https://github.com/WICG/turtledove/issues/632) for more information."